requirements.txt

```diff
@@ -1,3 +1,4 @@
 zeroconf==0.17.5
 requests>=2.0
 netifaces>=0.10.0
+orvibo==1.1.1
```

Comment:
I don't want to add a dependency that is meant to discover a single type of device. Only generic network like dependencies should be included.